pytorch_lightning/strategies/fully_sharded.py

```diff
@@ -205,9 +340,6 @@ def predict_step(self, *args, **kwargs) -> STEP_OUTPUT:
         with self.precision_plugin.predict_step_context():
             return self.model.predict_step(*args, **kwargs)
 
-    def post_training_step(self):
-        pass
-
```

Comment:
pre_backward is missing too. it was implemented in ddp but the regular sharded strategy also ignores it. 
asking for confirmation that this is intentional

```py
    def pre_backward(self, closure_loss: torch.Tensor) -> None:
        """Run before precision plugin executes backward."""
        if not self.lightning_module.automatic_optimization:
            prepare_for_backward(self.model, closure_loss)
```